index.js

```diff
@@ -209,36 +209,41 @@ module.exports = function (opts) {
         done(err)
       } else {
         // Render all individual content.
-        var contentFiles = minimatch.match(filesKeys, opts.pattern, {matchBase: true})
-        async.map(contentFiles, processFile, function (err) {
-          if (err) {
-            done(err)
-          } else {
-            // Render the content within the layouts.
-            async.map(contentFiles, renderContent, function (err) {
-              if (err) {
-                done(err)
-              } else {
-                // Delete the layout data.
-                async.map(layouts, deleteFile, function (err) {
-                  if (err) {
-                    done(err)
-                  } else {
-                    // Now rename all the files.
-                    async.map(Object.keys(files), renameFile, function (err) {
-                      if (err) {
-                        done(err)
-                      } else {
-                        // Now delete auxiliary metadata from files.
-                        async.map(Object.keys(files), deleteAuxiliaryMetadata, done)
-                      }
-                    })
-                  }
-                })
-              }
-            })
-          }
-        })
+        try {
+          var contentFiles = minimatch.match(filesKeys, opts.pattern, {matchBase: true})
```

Comment:
haha, it was just laziness on my part, really - I've been battling with contentful, metalsmith-layouts, and metalsmith-jstransformer for 2 days, so my patience was shot until I solved the problem a few minutes ago. I admit, that's pretty horrible. /ashamed

I originally left the async call outside the try catch, but the linter then complained.

How about this:

```js
        var contentFiles = []

        try {
          contentFiles = minimatch.match(filesKeys, opts.pattern, {matchBase: true})
        } catch (err) {
          done(err)
        }

        async.map(contentFiles, processFile, function (err) {
          ..

```